react/ReadMore/ReadMore.less

```diff
@@ -8,13 +8,30 @@
 }
 
 .fadeOut {
-  background: linear-gradient(to bottom, rgba(255, 255, 255, 0) 0%, white 100%);
   position: absolute;
   height: (@row-height * 4);
   width: 100%;
   bottom: 0;
 }
 
+.card {
+  background:
+    linear-gradient(
+      to bottom,
+      fade(@sk-card-bg, 0) 0%,
+      @sk-card-bg 100%
+    );
+}
+
+.body {
+  background:
+    linear-gradient(
+      to bottom,
+      fade(@sk-background, 0) 0%,
```

Comment:
should we do a background mixin with color argument here so we do not duplicate it?